n6001/hardware/ofs_n6001/board_spec.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <board version="22.4" name="ofs_n6001">
 
-  <compile name="afu_flat" project="ofs_top" revision="afu_flat" qsys_file="none" generic_kernel="1">
```

Comment:
No need to have an afu_flat revision at the beginning; this step can be done using the default ofs_pr_afu.qsf revision.